From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Mon, 23 Apr 2012 15:59:17 -0700
Subject: [021/106] Staging: line6: remove unneeded initialization

commit 305b8766a9833491b388ac98d57ce38a8106a1e0 upstream.

Static variables are initialized to NULL, no need to do it again in the
module_init function.

CC: Markus Grabner <grabner@icg.tugraz.at>
CC: Stefan Hajnoczi <stefanha@gmail.com>
CC: Julia Lawall <julia@diku.dk>
CC: Dan Carpenter <dan.carpenter@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/staging/line6/driver.c |    5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/drivers/staging/line6/driver.c b/drivers/staging/line6/driver.c
index e8023af..312905a 100644
--- a/drivers/staging/line6/driver.c
+++ b/drivers/staging/line6/driver.c
@@ -1297,13 +1297,10 @@ static struct usb_driver line6_driver = {
 */
 static int __init line6_init(void)
 {
-	int i, retval;
+	int retval;
 
 	printk(KERN_INFO "%s driver version %s\n", DRIVER_NAME, DRIVER_VERSION);
 
-	for (i = LINE6_MAX_DEVICES; i--;)
-		line6_devices[i] = NULL;
-
 	retval = usb_register(&line6_driver);
 
 	if (retval) {
